{"schemaVersion":{"patch":0,"major":0,"minor":3},"variants":[{"paths":["\/documentation\/atprotokit\/atrequestprepareerror\/failedafterretries"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATRequestPrepareError\/failedAfterRetries"},"sections":[],"kind":"symbol","abstract":[{"type":"text","text":"The request failed after several retries."}],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"failedAfterRetries"}],"languages":["swift"]}],"kind":"declarations"}],"metadata":{"title":"ATRequestPrepareError.failedAfterRetries","modules":[{"name":"ATProtoKit"}],"roleHeading":"Case","role":"symbol","externalID":"s:10ATProtoKit21ATRequestPrepareErrorO18failedAfterRetriesyA2CmF","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"failedAfterRetries","kind":"identifier"}],"remoteSource":{"fileName":"ATProtoError.swift","url":"https:\/\/github.com\/MasterJ93\/ATProtoKit\/tree\/main\/Sources\/ATProtoKit\/Errors\/ATProtoError.swift#L339"},"symbolKind":"case"},"hierarchy":{"paths":[["doc:\/\/ATProtoKit\/documentation\/ATProtoKit","doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATProtoError","doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATRequestPrepareError"]]},"references":{"doc://ATProtoKit/documentation/ATProtoKit":{"role":"collection","images":[{"type":"icon","identifier":"atprotokit_icon.png"}],"identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit","title":"ATProtoKit","url":"\/documentation\/atprotokit","type":"topic","kind":"symbol","abstract":[{"text":"Develop and manage client and server-side applications for the AT Protocol and Bluesky.","type":"text"}]},"doc://ATProtoKit/documentation/ATProtoKit/ATProtoError":{"title":"ATProtoError","kind":"symbol","identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATProtoError","type":"topic","url":"\/documentation\/atprotokit\/atprotoerror","fragments":[{"kind":"keyword","text":"protocol"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ATProtoError"}],"navigatorTitle":[{"kind":"identifier","text":"ATProtoError"}],"role":"symbol","abstract":[{"type":"text","text":"The base exception class for ATProtoKit."}]},"doc://ATProtoKit/documentation/ATProtoKit/ATRequestPrepareError":{"navigatorTitle":[{"kind":"identifier","text":"ATRequestPrepareError"}],"kind":"symbol","type":"topic","title":"ATRequestPrepareError","identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATRequestPrepareError","abstract":[{"type":"text","text":"An error type related to issues surrounding preparing a request to be sent."}],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ATRequestPrepareError","kind":"identifier"}],"url":"\/documentation\/atprotokit\/atrequestprepareerror"},"doc://ATProtoKit/documentation/ATProtoKit/ATRequestPrepareError/failedAfterRetries":{"role":"symbol","title":"ATRequestPrepareError.failedAfterRetries","abstract":[{"text":"The request failed after several retries.","type":"text"}],"type":"topic","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failedAfterRetries","kind":"identifier"}],"kind":"symbol","url":"\/documentation\/atprotokit\/atrequestprepareerror\/failedafterretries","identifier":"doc:\/\/ATProtoKit\/documentation\/ATProtoKit\/ATRequestPrepareError\/failedAfterRetries"},"atprotokit_icon.png":{"type":"image","identifier":"atprotokit_icon.png","alt":"A technology icon representing the ATProtoKit framework.","variants":[{"url":"\/images\/atprotokit_icon.png","traits":["1x","light"]}]}}}